Formula/cereal.rb

```diff
@@ -14,6 +13,9 @@ class Cereal < Formula
     sha256 "00ef7732975fb675326d0dc1f1a8732b995626075cb5f0a3f2740f28c3fcdda6" => :mountain_lion
   end
 
+  # error: chosen constructor is explicit in copy-initialization
+  patch :DATA if MacOS.version <= :mavericks
```

Comment:
No, and I'm not confident in the fix, so I'd prefer to just drop Mavericks support here unless Misty agrees this fix is sane.